src/miner.h

```diff
@@ -165,7 +165,7 @@ class BlockAssembler
 public:
     BlockAssembler(const CChainParams& chainparams);
     /** Construct a new block template with coinbase to scriptPubKeyIn */
-    std::unique_ptr<CBlockTemplate> CreateNewBlock(const CScript& scriptPubKeyIn);
+    std::unique_ptr<CBlockTemplate> CreateNewBlock(const CScript& scriptPubKeyIn, bool TestBlock = false);
```

Comment:
Do we need a default value here?
Do we really save that much disruption by not passing false explicitly where it's needed?

Bikeshedding: s/TestBlock/fValidateBlock/ ?